pkg/reading/service_read_issue.go

```diff
@@ -71,3 +82,25 @@ func (s *ServiceReadIssue) PostAPICall(result interface{}) error {
 func (s *ServiceReadIssue) Name() string {
 	return "ServiceReadIssue"
 }
+func (s *ServiceReadIssue) ExecuteAsLastStep(ctx configuration.Context) error {
+	if file, err := result.CreateDestination(s.destination); err != nil {
+		return err
+	} else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4860757)